jiant/tasks/tasks.py

```diff
@@ -3731,3 +3732,125 @@ def get_metrics(self, reset=False):
         """Get metrics specific to the task"""
         acc = self.scorer1.get_metric(reset)
         return {"accuracy": acc}
+
+
+@register_task("wikipedia_corpus_sop", rel_path="wikipedia_corpus_small")
+class SentenceOrderTask(Task):
+    """ Task class for Sentence Order Prediction (SOP) with Wikipedia data (more specifically,
+        the dump from Wikimedia). See the ALBERT paper for details on SOP: https://arxiv.org/abs/1909.11942.
+        We are currently using an unpreprocessed version of the Wikipedia corpus
```

Comment:
Please fix the comment on this from the earlier PR>

Reply:
This seems like a contradiction—if it's unpreprocessed, why do we have a data preparation script.

Reply:
Also, like we talked about, it should be clear exactly what dataset you get when you run these scripts. Make it clear somewhere (maybe in the scripts readme) whether you're getting a specific dump, or just whatever is most recent.

Reply:
I've updated it.